lib/core/renderer.js

```diff
@@ -532,8 +532,11 @@ export default class Renderer extends Tapable {
       resourceHints = context.renderResourceHints()
       HEAD += resourceHints
     }
-    APP += `<script type="text/javascript">window.__NUXT__=${serialize(context.nuxt, { isJSON: true })};</script>`
-    APP += context.renderScripts()
+
```

Comment:
We are working to improve renderer soon. So meanwhile I think it would be better to not making changes into `renderer.js` or making it in a separate PR. 
Also Thanks for your works :)

Reply:
Same as @pi0 , I don't think it's a good idea at all to add this condition.

Reply:
@pi0 @Atinux Completely understand. Suggestions for alternatives for satisfying the AMP requirement related to no `<script>` tags in the body when using nuxt.js for SSR?